scripts/us_fda/drugs/create_enums.py

```diff
@@ -0,0 +1,453 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Contains functions involving creating enum schema mcf files and loading
+enum to dicd dictionaries used in cleaning and writing the data mcf.
+
+Calling create_enums.py from command line will write enum schema to
+FDADrugsEnumSchema.mcf
+"""
+import pandas as pd
+
+
+def add_to_dict_not_set(dictionary, key, value):
+    """Write key value pair to dictionary
+
+    Enum to dcid dictionaries should be 1:1, if multiple values(dcids) attempt
+    to be added to the same key(enum), an error message will be printed.
+    """
+    if not value or pd.isnull(value):
+        print("Null value added to dict, key: " + key)
+    if key not in dictionary:
+        dictionary[key] = value
+    else:
+        print('tried to overwrite key: ' + key)
+
+
+def parse_dosage_form_fda(enum_schema_file, df_dict, row):
+    """Create a Dosage Form enum from a row of the dosage_forms_fda dataframe
+
+    The enum dcid is loaded into df_dict which stores the enum to dcid
+    relationship.
+    If enum_schema_file is not None, then the enum is written in mcf format to
+    the file
+    """
+    is_sub_enum = False
+
+    dosage_form_name = str(row['Term']).title()
+    code = str(row['Code'])
+
+    dosage_form_name = dosage_form_name.replace(' / ', ', ')
+    dosage_form_name = dosage_form_name.replace('/ ', ', ')
+    dosage_form_name = dosage_form_name.replace('/', ',')
+    dosage_form_name = dosage_form_name.replace('Pellets', 'Pellet')
+
+    split = dosage_form_name.split(',')
+    split_length = len(split)
```

Comment:
nit: skip the variable (it overall adds more letters and lines to code without clear benefit over just using len(split))